packages/editor/src/client/blocks/richTextBlock/editor/elements.ts

```diff
@@ -0,0 +1,34 @@
+import {BlockFormat} from './formats'
+import {Element as SlateElement} from 'slate'
+
+export function emptyTextParagraph(): SlateElement {
```

Comment:
There is already a function called `createDefaultValue()` in the file `packages/editor/src/client/blocks/richTextBlock/richTextBlock.tsx`

Reply:
Did it the other way round: `createDefaultValue() { return emptyTextParagraph ... `